features/pqs_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'feature_helper'
+
+feature 'Parliamentary Question view', js: true, suspend_cleaner: true do
+  include Features::PqHelpers
+
+  before(:all) do
+    DBHelpers.load_feature_fixtures
+    @pqs = PQA::QuestionLoader.new.load_and_import(3)
+  end
+
+  after(:all) do
+    DatabaseCleaner.clean
+  end
+
+
+  context 'viewing a non-existent uin produces 404' do
+	  scenario 'GET show' do
+	    create_pq_session
+	    visit pq_path(invalid_uin)
+	    expect(page.status_code).to eq 404
+	    expect(page).to have_content("The page you were looking for doesn't exist. (Error 404)")
+	  end
+	  
+	end
+
+end
+
+
+def invalid_uin
```

Comment:
Ok, if we can be sure that the test data will only ever have uins 1-3, then something much simpler will work.  Have updated it.